src/ddmd/clone.d

```diff
@@ -1008,6 +1010,13 @@ extern (C++) FuncDeclaration buildDtor(AggregateDeclaration ad, Scope* sc)
         auto sdv = (cast(TypeStruct)tv).sym;
         if (!sdv.dtor)
             continue;
+        // fix: https://issues.dlang.org/show_bug.cgi?id=17257
+        {
```

Comment:
What are the braces here for? Some comment about what is going on would be nice.